### ISSUE TITLE
Strange Artifacts While Driving In Cyberpunk 2077

### ISSUE THREAD

**Issue by HayTTG** [Nukem9/dlssg-to-fsr3#39]
Noticed when you drive around. the bottom quarter of the screen chops up and shows double of things. Only while FG is enabled, so this must be a bug.

**Comment by Dravic** [comment]
Yes, this is 100% a problem for me as well and is strictly caused by flipping Frame Generation ON.
Turning it off removes this issue.

More importantly this seems to be tied to your traversal speed. It doesn't seem to be pronounced when standing still or moving slowly.
